Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection.Metadata;
+using System.Runtime.CompilerServices;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static float IsNumber(string input)
+        {
+            var isEmpty = String.IsNullOrEmpty(input);
```

Comment:
You can probably inline this with the line bellow- little to do still is reasonably readable.